.github/workflows/stable.yml

```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        version: [9.11.0]
+        version: [9.11.0, 9.12.0]
```

Comment:
lets drop 9.11

Reply:
I've found another issue #3. It might be interesting to fix also 9.11.

Reply:
Do you want me to open another pull request for #3 and then remove version 9.11 from this one?